src/meson.build

```diff
@@ -70,4 +70,10 @@ if shellcheck.found()
 endif
 
 test('go fmt', meson_go_fmt_program, args: [meson.current_source_dir()])
-test('toolbox go unit tests', go, args: ['test', '-v', './...'], workdir: meson.current_source_dir())
+test(
+  'toolbox go unit tests',
+  go,
+  args: ['test', '-v', './...'],
+  env: 'LC_ALL=C',
```

Comment:
Does it really have to be the `C` locale?  Can't we use an UTF-8 locale like `en_US.UTF-8`?

I am saying this because for quite a long time, GNOME Terminal [doesn't even start](https://wiki.gnome.org/Apps/Terminal/FAQ#Exit_status_8) without a UTF-8 locale.  So, a non-UTF-8 locale isn't very representative of the environment in which Toolbx is likely to be used.

Reply:
I agree that C shouldn't be used